gstd/Cargo.toml

```diff
@@ -21,6 +21,10 @@ futures = { workspace = true, features = ["alloc"] }
 static_assertions.workspace = true
 
 [features]
-default = ["panic-handler"]
+default = ["default-stable", "nightly"]
+default-stable = ["panic-handler"]
```

Comment:
I don't understand why this feature is needed

Reply:
That would be a breaking change, bwcause users of this crate will no longer get OOM handler and verbose panic messages by default. Prior to this change, crate wouldn't compile on stable, so changing defaults for stable users isn't a braking change.

Reply:
On the other hand, any codebase using nightly features is prone to breaking due to Rust upgrades, but that should be considered separately IMHO.

Reply:
> so changing defaults for stable users isn't a breaking change.

> any codebase using nightly features is prone to breaking due to Rust upgrades

Agree

> but that should be considered separately IMHO.

I think this is implied by default like it is in nightly toolchain and crates with features like  `nightly` or `unstable`

Reply:
Well, I see two options:

- Break everything with this PR right now (breakage has a big chance to appear unnoticed until the user opens the logs and finds out they are almost empty; expect issues on "Why are my logs empty?"), immediately affects everyone
- Do not break anything (if the nightly features used become stabilized, compiler will emit a warning, but nothing more than that)

I do not think "nightly can break with updates sometimes" is enough for making actual breaking changes with no motivation behind.